install.sh

```diff
@@ -198,6 +187,7 @@ ln -sfv "../lib/crew/bin/crew" "${CREW_PREFIX}/bin/"
 # prepare sparse checkout .rb packages directory and do it
 cd "${CREW_LIB_PATH}"
 rm -rf .git
+git config --global init.defaultBranch main
```

Comment:
Wouldn't the default branch be master?

Reply:
This gets rid of the warning message, and we're pulling from master. The local branch name can be anything. (Do we hard code that local branch name anywhere?)

Reply:
Doing this would require us to execute `git pull origin master` rather than just `git pull`, correct?

Reply:
changed git init to clone so this line is no longer needed.